packages/verification-vue-component/src/components/Verification/items/cards/FaqCard.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+    <BaseCard :card-class="'faq-card'">
+        <template #header>
+            <div class="title">
+                <span class="help-card-title">{{ _$t('verification.card.faq.title') }}</span>
+            </div>
+        </template>
+        <template #body>
+            <div class="faq-items">
+                <div class="item"
+                     v-for="(item, index) in _$t('verification.card.faq.items', { returnObjects: true })"
+                     :key="index">
+                    <input type="checkbox" :id="'chck' + index">
+                    <label class="item-label" :for="'chck' + index">{{ item.title }}</label>
+                    <div class="item-content">
+                        {{ item.content }}
+                    </div>
+                </div>
+            </div>
+        </template>
+        <template #footer>
+            <div class="left">
+                <button class="btn secondary" @click="toggleHelp('faq')">
```

Comment:
This button has a margin-left which I guess, shouldn't be there.

Reply:
Yes, fixed that already in my new branch but did it here now as well